docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# This file proves to be less useful if the url of the server never changes and all tests are always run.
```

Comment:
I think it would be nice to start with a sentence or two on why the/an entry point script can be useful and how it's used rather than with its limitations.

It might also perhaps be better to add a section to the `README.md` explaining the build and the entry point script, rather than adding it as comments here.

Reply:
added a section in the readme - let me know what you think!